src/getting_grain.md

```diff
@@ -2,16 +2,60 @@
 title: Getting Grain
 ---
 
-## Building Grain
+We have a couple different ways to get Grain. Most developers will prefer the Packaged Grain binaries, but make sure to read the note about speed!
 
-Grain currently only works on Mac and Linux. We are hard at work trying to integrate Binaryen and OCaml on Windows, but this is uncharted territory! Please let us know if you have this low-level expertise.
+## Packaged Grain
 
-To build the compiler, you'll need [Node.js](https://nodejs.org/en/download/current/) v14 and [Yarn](https://yarnpkg.com/getting-started/install).
+The Grain toolchain (including our CLI, compiler, runtime, and standard library) is shipped as a single binary. Binaries are available for [MacOS x64](#MacOS-x64---Homebrew), [Linux x64](#Linux-x64---Download), and [Windows x64](#Windows-x64---Download).
+
+__Note:__ These binaries are a bit slow the first time you build a project (around 70 seconds) because they build and output the runtime & standard library into your project. If you need extra speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
```

Comment:
```suggestion
__Note:__ These binaries are a bit slow the first time you build a project (around 70 seconds). The reason for this is that the compiler is actually running in JavaScript, _and_ it is building and writing the runtime & standard library into your project. If you need extra speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
```

@ospencer What do you think about this mentioning it is a JS compiler and then linking to the Building from Source section which mentions native compiler.

Reply:
We decided this on Discord:
```suggestion
__Note:__ These binaries are a bit slow when first building a project (around 70 seconds). The packaged compiler is running in JavaScript, _and_ it builds and writes the runtime & standard library into your project. If you need raw speed, you can build the native compiler from source! See [Building Grain from Source](#Building-Grain-from-Source) below.
```